Userland/Libraries/LibCore/SecretString.cpp

```diff
@@ -1,10 +1,14 @@
 /*
  * Copyright (c) 2021, Brian Gianforcaro <bgianf@serenityos.org>
+ * Copyright (c) 2021, Mustafa Quraish <mustafa@cs.toronto.edu>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #include <LibCore/SecretString.h>
+#if defined(AK_OS_MACOS)
```

Comment:
We should probably include `AK/Platform.h` explicitly, so there's no ambiguity about if this will be defined or not?

Reply:
Will do, didn't consider that.